src/pages/blog/ironic-airship2-case-study.md

```diff
@@ -0,0 +1,81 @@
+---
+templateKey: blog-post
+title: Airship 2.0 - How Ironic Delivers Abstraction and Automation Using Open Source Software
+author: Alexander Hughes
+date: 2020-07-22T12:00:00.000Z
+category: 
+  - label: Airship 2.0
+    id: category-C98iZYrE1
+---
+
+## **ANNOUNCING THE IRONIC BAREMETAL WHITEPAPER**
+
+This week the OpenStack Foundation hosted the [_Hardware Automation_](
```

Comment:
If we can edit the name of the event to "OpenDev: Hardware Automation" that'd be great!

Reply:
Done!